collectd-openstack/README.md

```diff
@@ -52,6 +52,12 @@ Reference for OpenStack [Monitoring](https://wiki.openstack.org/wiki/Operations/
 
 ### REQUIREMENTS AND DEPENDENCIES
 
+#### Deployment Host
+
+Identify a host on which the SignalFx agent will run. This integration collects data from OpenStack remotely via APIs, and so those API endpoints must be visible to the host on which the agent runs.  We do not recommend installing the agent directly on a compute instance because,
```

Comment:
I don't think the numbers work. So I would just do... because a compute instance/resource under one project cannot get stats about the resources under other projects. Also, a compute instance may go down due to lack of resources in the project.

Reply:
updated